client/src/app/components/Projects/Projects.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+import "./projects.css";
+import ProjectTile from "../ProjectTile/ProjectTile";
+
+const projects = [
+  {
+    projectTitle: "Coding for Fun",
```

Comment:
I'd say we don't need the project prefix

Reply:
It should match the structure of the projects model in mongoose

Reply:
Updated